tests/robot-tests/README.md

```diff
@@ -179,12 +180,40 @@ IntelliJ also allows you use to External Tools to right click on a file and run
 - Working directory: `$ProjectFileDir$` (which should represent the robot-tests directory. You may have to do something like `$ProjectFileDir$/../tests/robot-tests`)
 
 
+Alertatively you can use Pycharm CE (https://www.jetbrains.com/pycharm/download/#section=windows) as an IDE and follow the above steps to enable External Tools. 
```

Comment:
Typo: Alertatively -> Alternatively

I think you also only need one line break above it.